data/distrodefs/fedora/imagetypes.yaml

```diff
@@ -752,6 +752,91 @@
       aarch64:
         <<: *iot_simplified_installer_partition_tables_x86
 
+  supported_options_lists:
+    # common options supported by all disk image types this includes everything
+    # that is not specific to installers or ostree-based images
+    supported_options_disk: &supported_options_disk
+      - "name"
+      - "version"
+      - "description"
+      - "distro"
+      - "packages"
+      - "modules"
+      - "groups"
+      - "containers"
+      - "minimal"
+      - "customizations.cacerts"
+      - "customizations.directories"
+      - "customizations.disk"
+      - "customizations.files"
+      - "customizations.filesystem"
+      - "customizations.fips"
+      - "customizations.firewall"
+      - "customizations.user"
+      - "customizations.group"
+      - "customizations.hostname"
+      - "customizations.kernel"
+      - "customizations.locale"
+      - "customizations.openscap"
+      - "customizations.repositories"
+      - "customizations.rpm"
+      - "customizations.services"
```

Comment:
This is missing `customizations.sshkey` and `customizations.partition_mode` which I stumbled into by accident when trying a blueprint I had laying around from something else, not through any thorough examination so there may be others :/

Reply:
The former only exists in `osbuild-composer` and (AFAIK) gets translated to a `customizations.user` we mention it as [deprecated](https://osbuild.org/docs/user-guide/blueprint-reference/#ssh-keys) in the documentation though I've gotten bitten by it before as well.

The latter is indeed missing and [documented to exist](https://osbuild.org/docs/user-guide/blueprint-reference/#partitioning-mode).